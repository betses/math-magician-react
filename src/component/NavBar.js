import React from "react";
import "../style/Navbar.css";
import { Link } from "react-router-dom";

const NavBar = () => (
  <div className="nav-bar">
    <h1>Math Magician</h1>
    <ul className="nav-items">
      <li>
        <Link to="/" className="item">
          Home
        </Link>
      </li>
      <li>
        <Link to="/calculator" className="item">
          Calculator
        </Link>
      </li>
      <Link to="/quote" className="item">
        quote
      </Link>
    </ul>
  </div>
);

export default NavBar;
