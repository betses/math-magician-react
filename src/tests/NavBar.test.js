import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import NavBar from "../component/NavBar";

it("Render the page correctluy", () => {
  const tree = renderer
    .create(
      <Router>
        <NavBar />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
