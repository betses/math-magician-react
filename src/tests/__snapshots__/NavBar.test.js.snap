// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the page correctluy 1`] = `
<div
  className="nav-bar"
>
  <h1>
    Math Magician
  </h1>
  <ul
    className="nav-items"
  >
    <li>
      <a
        className="item"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="item"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <a
      className="item"
      href="/quote"
      onClick={[Function]}
    >
      quote
    </a>
  </ul>
</div>
`;
